yellowbrick/base.py

```diff
@@ -89,6 +89,21 @@ def ax(self):
     def ax(self, ax):
         self._ax = ax
 
+    @property
+    def size(self):
+        """
+        Returns the actual size in pixels as set by matplotlib, or
+        the user provided size if available.
+        """
+        if not hasattr(self, "_size") or self._size is None:
+            fig = plt.gcf()
+            self._size = fig.get_size_inches()*fig.dpi
+        return self._size
+
+    @size.setter
+    def size(self, size):
+        self._size = size
```

Comment:
@gary-mayfield this is great thank you so much! And this is certainly a complete PR, however I'm wondering if you'd like to extend it a bit in the setter? 

Basically right now, setting size doesn't actually change the size of the matplotlib figure, it just changes a property that would eventually get used by the `poof()` method. I propose that instead you do actually change the size of the figure as in [this stack overflow post](https://stackoverflow.com/questions/332289/how-do-you-change-the-size-of-figures-drawn-with-matplotlib) 

There is a `fig.set_size_inches` method, perhaps there is also a `fig.set_size_pixels` method? Or you could back out the pixels with the DPI function. 

I've approved the PR, so if you'd prefer to just merge this work in; then that is fine with me; if you're interested in doing the extension let me know!

Reply:
@bbengfort I'll have a go at the proposed extension and let you know what I come up with.